node/tests/PubSub.test.ts

```diff
@@ -570,25 +640,21 @@ describe("PubSub", () => {
 
                 // Check if all messages are received correctly
                 for (let index = 0; index < NUM_CHANNELS; index++) {
-                    const pubsubMsg = await getMessageByMethod(
+                    const pubsubMsg = (await getMessageByMethod(
                         method,
                         listeningClient,
                         context,
                         index,
-                    );
+                    ))!;
```

Comment:
why did you add this?

Reply:
VS Code complaints that `pubsubMsg` is possibly `null`. So I have to add this.